lib/pagseguro/payment_request.rb

```diff
@@ -82,17 +85,21 @@ def shipping=(shipping)
 
     # Calls the PagSeguro web service and register this request for payment.
     def register
-      path = if @receivers.empty?
-               'checkout'
-             else
-               'checkouts'
-             end
+      request = if @receivers.empty?
+                  Request.post('checkout', api_version, params)
+                else
+                  Request.post_xml('checkouts', api_version, credentials, xml_params)
```

Comment:
I understand. In my case I though do this way, but I sent only xml and delivery. I'm waiting for the test and I hope it passes! ;P